File: app/src/main/java/com/example/infoappv2/StartScreenActivity.java
package com.example.infoappv2;

import android.content.Intent;
import android.os.Bundle;
import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;
import java.util.Timer;

public final class StartScreenActivity extends AppCompatActivity {

        protected void onCreate(@Nullable Bundle savedInstanceState) {
                super.onCreate(savedInstanceState);
                this.setContentView(R.layout.activity_startscreen);
                this.openInfoActivity();
        }

        private void openInfoActivity() {
                Timer var1 = new Timer();
                var1.schedule(getApplicationContext(startActivity(Intent())), 3000);
        }
}

File: app/src/main/java/com/example/infoappv2/StartScreenActivity.kt
package com.example.infoappv2

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import java.util.*
import kotlin.concurrent.schedule

class StartScreenActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_startscreen)
        openInfoActivity()
    }

    private fun openInfoActivity() {
        Timer().schedule(3000){
             Intent(applicationContext, InfoActivity::class.java).also {
                 startActivity(it)
             }
             }
    }
}